src/server/saved_objects/saved_objects_mixin.js

```diff
@@ -30,30 +30,34 @@ export function savedObjectsMixin(kbnServer, server) {
     const adminCluster = server.plugins.elasticsearch.getCluster('admin');
 
     try {
-      await adminCluster.callWithInternalUser('cluster.health', {
-        timeout: `${server.config().get('savedObjects.indexCheckTimeout')}ms`,
-        index: server.config().get('kibana.index'),
-        waitForStatus: 'yellow',
+      const index = server.config().get('kibana.index');
+      await adminCluster.callWithInternalUser('indices.putTemplate', {
+        name: `kibana_index_template:${index}`,
+        body: {
+          template: index,
+          settings: {
+            number_of_shards: 1
+          },
+          mappings: server.getKibanaIndexMappingsDsl()
+        }
       });
     } catch (error) {
-      // This check is designed to emulate our planned index template move until
-      // we get there, and once we do the plan is to just post the index template
-      // and attempt the request.
-      //
-      // Because of this we only throw NotFound() when the status is red AND
-      // there are no shards. All other red statuses indicate real problems that
-      // will be described in better detail when the write fails.
-      if (
-        error &&
-        error.body &&
-        error.body.status === 'red' &&
-        !error.body.unassigned_shards &&
-        !error.body.initializing_shards &&
-        !error.body.delayed_unassigned_shards
-      ) {
-        server.log(['debug', 'savedObjects'], `Attempted to write to the Kibana index when it didn't exist.`);
-        throw new adminCluster.errors.NotFound();
-      }
+      server.log(['debug', 'savedObjects'], {
+        tmpl: 'Attempt to write indexTemplate for SavedObjects index failed: <%= err.message %>',
+        es: {
+          resp: error.body,
+          status: error.status
+        },
+        err: {
+          message: error.message,
+          stack: error.stack,
```

Comment:
__nit:__ let's be consistent on the trailing commas within this file  :)

Reply:
So ready for https://github.com/elastic/kibana/pull/14631#issuecomment-341822569